FidelLimited/docs
# [CLET-291][MINOR] 3DS Section on Web SDK Docs

#### Steps to test

- Go to the website project.
- Run `env DOCS_BRANCH=feat/CLET-291-sdk-section-3ds yarn docs` to download the docs files from this PR.
- Run `yarn start` and go to `/docs` to check the changes.
